packages/typescript-typings/types/web3-provider-engine/index.d.ts

```diff
@@ -1,10 +1,18 @@
 declare module 'web3-provider-engine' {
+    interface Web3ProviderEngineOptions {
+        pollingInterval?: number;
+        blockTracker?: any;
+        blockTrackerProvider?: any;
```

Comment:
Any chance you could also provider the types for these rather then `any`?

Reply:
We don't have an interface for the provider. I think it different issue to implement this interface.

Reply:
I agree. I soon will be migrating those types to `DefinitelyTyped` and can look into that then.